src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search__form {
+  display: inline-block;
```

Comment:
Do you really need to set `inline-block` here? It's a block by default and I don't see any need in `inline-block` here